tests/DoctrineORMModuleTest/CliTest.php

```diff
@@ -81,65 +90,145 @@ public function testValidHelpers()
+
+        $module = new Module();
+        $module->initializeConsole($event);
+
+        /* @var $entityManagerHelper \Doctrine\ORM\Tools\Console\Helper\EntityManagerHelper */
+        $entityManagerHelper = $application->getHelperSet()->get('entityManager');
+        $this->assertInstanceOf('Doctrine\ORM\Tools\Console\Helper\EntityManagerHelper', $entityManagerHelper);
+        $this->assertSame($this->entityManager, $entityManagerHelper->getEntityManager());
+    }
+
+    /**
+     * @backupGlobals enabled
+     */
+    public function testEntityManagerUsedCanBeSpecifiedInCommandLineArgument()
+    {
+        $connection = $this->getMockBuilder('Doctrine\DBAL\Connection')
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $entityManager = $this->getMockbuilder('Doctrine\ORM\EntityManager')
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $entityManager
+            ->expects($this->atLeastOnce())
+            ->method('getConnection')
+            ->willReturn($connection);
+
+        $this->serviceManager->setService('doctrine.entitymanager.some_other_name', $entityManager);
+
+        $application = new Application();
+        $event = new Event('loadCli.post', $application, ['ServiceManager' => $this->serviceManager]);
+
+        $_SERVER['argv'][] = '--object-manager=doctrine.entitymanager.some_other_name';
+
+        $module = new Module();
+        $module->initializeConsole($event);
+
+        /* @var $entityManagerHelper \Doctrine\ORM\Tools\Console\Helper\EntityManagerHelper */
+        $entityManagerHelper = $application->getHelperSet()->get('entityManager');
+        $this->assertInstanceOf('Doctrine\ORM\Tools\Console\Helper\EntityManagerHelper', $entityManagerHelper);
+        $this->assertSame($entityManager, $entityManagerHelper->getEntityManager());
+    }
+
+    /**
+     * @param string $commandName
+     * @param string $className
+     *
+     * @dataProvider dataProviderForTestValidCommands
+     */
+    public function testValidCommands($commandName, $className)
+    {
+        /* @var $command \Symfony\Component\Console\Command\Command */
+        $command = $this->cli->get($commandName);
+        $this->assertInstanceOf($className, $command);
+
+        // check for the entity-manager option
+        $this->assertTrue($command->getDefinition()->hasOption('object-manager'));
+
+        $entityManagerOption = $command->getDefinition()->getOption('object-manager');
+
+        $this->assertTrue($entityManagerOption->isValueOptional());
+        $this->assertFalse($entityManagerOption->isValueRequired());
+        $this->assertFalse($entityManagerOption->isArray());
+        $this->assertNull($entityManagerOption->getShortcut());
+        $this->assertSame('doctrine.entitymanager.orm_default', $entityManagerOption->getDefault());
+        $this->assertSame('The name of the object-manager to use.', $entityManagerOption->getDescription());
```

Comment:
`object-manager` is not hyphenated in sentence context.  Please change to `object manager`